Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/crash.ui/SettingsCrashReportViewModel.kt

```diff
@@ -0,0 +1,44 @@
+package de.rki.coronawarnapp.test.crash
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.viewModelScope
+import com.squareup.inject.assisted.AssistedInject
+import de.rki.coronawarnapp.bugreporting.event.BugEvent
+import de.rki.coronawarnapp.bugreporting.reportProblem
+import de.rki.coronawarnapp.bugreporting.storage.repository.BugRepository
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModel
+import de.rki.coronawarnapp.util.viewmodel.SimpleCWAViewModelFactory
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import timber.log.Timber
+import java.lang.Exception
+
+class SettingsCrashReportViewModel @AssistedInject constructor(
+    private val crashReportRepository: BugRepository
+) : CWAViewModel() {
+
+    val crashReports = crashReportRepository.getAll()
+
+    lateinit var selectedCrashReport: LiveData<BugEvent>
```

Comment:
This is just waiting for an `UninitializedPropertyAccessException` please solve this differently.